app/scripts/views/settings.js

```diff
@@ -148,6 +149,34 @@ function ($, modal, Cocktail, Session, BaseView, AvatarMixin,
       return self._showAvatar();
     },
 
+    beforeDestroy: function () {
+      $('body.settings').fadeOut(this.FADE_OUT_SETTINGS_MS, function (){
+        $('body').removeClass('settings').show();
+      });
+    },
```

Comment:
> Ugh, I didn't notice in 207dfd6 that we removed the smooth transition.
>
> /cc @shane-tomlinson

Yeah, that transition messed with the removal of the `layoutClassName`, so I took it out.